### PR TITLE
New board set-up for easier integration into JS

### PR DESCRIPTION
**Pull request by TMUND**

### CODE REVIEW

File: ticktacotoe.sublime-project [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"folders":
+	[
+		{
+			"path": "/C/Users/Toni/Desktop/projects/ticktacotoe"
```
Comment (iangilman) [inline ticktacotoe.sublime-project:5]: You could also make this `"path": "."` which means "the directory this project file is in", which will work for other people as well (who don't have a Users/Toni directory).

File: ticktacotoe.sublime-workspace [hunk 2]
```diff
@@ -0,0 +1,318 @@
+{
```
Comment (iangilman) [inline ticktacotoe.sublime-workspace:1]: While the sublime project file is meant to be shared, the sublime workspace is best kept separate for each person (it keeps track of things like which tabs you have open). You can remove it from the repository and then add an entry to your `.gitignore` file so it doesn't get checked in when sublime re-creates it.

File: index.html [hunk 3]
```diff
@@ -8,34 +8,34 @@
...
-        <td onclick="sayHello();"class="sqr 7">X</td>
-        <td onclick="sayHello();"class="sqr 8 v">X</td>
-        <td onclick="sayHello();"class="sqr 9">X</td>
-      </tr>
-    </table>
-  </div>
-  <div>Hide and Show jquery example</div>
+    <div id="game-wrap">
+      
+      <div class="row">
```
Comment (iangilman) [inline index.html:13]: Nice! Looks cleaner. I look forward to seeing how it integrates with the JavaScript. The CSS looks good too.

**Comment by iangilman** [comment]
:shipit:

**Comment by TMUND** [comment]
Thanks for the tips, cleaning it up now!

**Pull request merged**